server/accounts.go

```diff
@@ -4074,31 +4078,42 @@ func (dr *DirAccResolver) Start(s *Server) error {
 		if len(tk) != accLookupReqTokens {
 			return
 		}
-		if theJWT, err := dr.DirJWTStore.LoadAcc(tk[accReqAccIndex]); err != nil {
-			s.Errorf("Merging resulted in error: %v", err)
+		accName := tk[accReqAccIndex]
+		if theJWT, err := dr.DirJWTStore.LoadAcc(accName); err != nil {
+			if errors.Is(err, fs.ErrNotExist) {
+				s.Debugf("DirResolver - Could not find account %q", accName)
+				go func() {
```

Comment:
So wondering if this should be a concern. I know why you are putting it into a separate go routine because of the sleep, but this could cause of an explosion of Go routines yes?

Like if we did a tight loop of clients trying to connect with a bad account we would be spinning these in every server for every lookup request no?

Reply:
Maybe what we do for this version of the server, is wait for all servers if we receive a nil response. This way we can send a nil response immediately but on the receiving side we see the nil so wait for more responses. We know how many other servers are in the system internally so we can know when we are done.

Reply:
sounds good, I will explore this idea and update. I was concerned about blocking but would now prefer to be cautious with spinning to many go routines here.

Reply:
Maybe do a quick debug.PrintStack() there too to see if we are rooted in a route or GW read..

If we are we can not block for sure, but this approach might work.

Reply:
I think that approach will work much better and can get faster responses for the happy path when servers online, I updated with this new heuristic.